src/components/sidebar/Sidebar.jsx

```diff
@@ -17,38 +17,44 @@ const Sidebar = () => {
         <div>
           <SidebarItem
             Item={HomeIcon}
-            type="Home"
+            type="/"
+            name="Home"
             active={active === 'Home'}
             clickHandler={setActive}
           />
           <SidebarItem
             Item={ExploreIcon}
             type="Explore"
+            name="Explore"
             active={active === 'Explore'}
             clickHandler={setActive}
           />
           <SidebarItem
             Item={LikedVideoIcon}
-            type="Likes"
+            type="/playlist/likedVideos"
+            name="Likes"
             active={active === 'Likes'}
             clickHandler={setActive}
           />
           <SidebarItem
             Item={HistoryIcon}
-            type="History"
+            type="/playlist/History"
+            name="History"
             active={active === 'History'}
             clickHandler={setActive}
           />
 
           <SidebarItem
             Item={PlaylistIcon}
             type="Playlist"
```

Comment:
In other tabs, you've given type as route and here you've given it a name I guess. You might want to correct that if not intentional.